spec/models/enrollment_export_spec.rb

```diff
@@ -48,29 +48,28 @@
 
     context "on_or_before" do
       let(:enrollment_export) { build(:enrollment_export) }
-
-      subject { enrollment_export.valid? }
```

Comment:
I can't see where the subject is being defined/called in the updated code

Reply:
this was just being used to trigger the validation (which just returns a true/false) and not being read.